lab5.cpp

```diff
@@ -0,0 +1,109 @@
+﻿#include <iostream>
+#include <string>
+#include <vector>
+using namespace std;
+class Guest
+{
+public:
+
+    int id;
+    string name;
+    int age;
+    string city;
+    int PhoneNumber;
+
+    int getId()
+    {
+        return id;
+    }
+    string getName()
+    {
+        return name;
+    }
+    int getAge()
+    {
+        return age;
+    }
+    string getCity()
+    {
+        return city;
+    }
+    int getPhoneNumber()
+    {
+        return PhoneNumber;
+    }
+
+    Guest(int id, string name, int age, string city, int PhoneNumber)
+    {
+        this->id = id;
+        this->name = name;
+        this->age = age;
+        this->city = city;
+        this->PhoneNumber = PhoneNumber;
```

Comment:
the name is not spelled correctly

Reply:
Don`t really ubderstand what do you mean